contracts/examples/introspection/erc165/IWhitelistId.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.5.10;
+import "./IWhitelist.sol";
+
+
+contract IWhitelistId {
```

Comment:
It's a contract with the ERC165 id of the `IWhitelist.sol` interface. I don't like the name either (because contracts prefixed by I are supposed to be interfaces). Do you have a suggestion?

Reply:
Meh, renamed by WhitelitInterfaceId. I don't like it much, but al least it is not as confusing.